### PR TITLE
Added recipe for Flask-FlatPages

### PR DESCRIPTION
**Pull request by pmlandwehr**

### CODE REVIEW

File: recipes/flask-flatpages/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  number: {{ build_num }}
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    # - flask >=0.8
+    # - pyyaml >=3.10
+    # - markdown >=2.3.1
```
Comment (ocefpaf) [inline recipes/flask-flatpages/meta.yaml]: Can you remove the commented out dependencies?

**Comment by jakirkham** [comment]
LGTM. Are you happy to merge this, @ocefpaf, or are there more changes needed?

**Comment by jakirkham** [comment]
Toggled to restart CIs.

**Comment by jakirkham** [comment]
Should note that this makes for an effective test of PR ( https://github.com/conda-forge/conda-forge-webservices/pull/53 ) to exclude merged recipes pending conversion from the lint. As we can see the CIs detect and [exclude `versiontools`]( https://travis-ci.org/conda-forge/staged-recipes/builds/161818646#L267 ) from the build. The linter would also have picked this up, but thanks to that exclusion PR it ignores it. Thus the lint remains the same regardless of what has been merged into `master`.

**Comment by jakirkham** [comment]
Thanks @pmlandwehr.

**Pull request merged**